src/panels/config/lovelace/resources/dialog-lovelace-resource-detail.ts

```diff
@@ -60,27 +65,34 @@ export class DialogLovelaceResourceDetail extends LitElement {
     fireEvent(this, "dialog-closed", { dialog: this.localName });
   }
 
+  private _closeDialog(): void {
```

Comment:
Make this the public function and the current public function the private one, so public close calls get nice animations too.